app/database/migrations/2014_07_24_082711_CreatekBLIStables.php

```diff
@@ -117,17 +117,30 @@ public function up()
             $table->softDeletes();
             $table->foreign('measure_id')->references('id')->on('measures');
         });
-
+        
+         Schema::create('panel', function(Blueprint $table)
+        {
```

Comment:
this need to be a new migration file. We are installing in already existing systems.